packages/epics/src/agreements/components/agreement-form-payment-schedule.tsx

```
@@ -0,0 +1,223 @@
+        defaultValues: {
+          option: 'Immediately',
+          futureDate: undefined,
+          milestones: [],
+        },
+        mode: 'onChange',
+      });
+
+  return { methods, isInsideProvider };
+}
+
+export function AgreementFormPaymentSchedule({}: AgreementFormPaymentScheduleProps) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)